src/components/collection/Authorship.js

```diff
@@ -80,7 +80,7 @@ const Authorship = ({ itemId, author, isLoading }) => {
               ) : (
                 <Avatar
                   useAvatar={hooks.useAvatar}
-                  alt={authorName}
+                  alt={t(`${authorName}'s avatar`)}
```

Comment:
What you could do is using variables! https://www.i18next.com/translation-function/interpolation